assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -141,15 +141,15 @@ storage:
       contents:
         inline: |
           KUBELET_IMAGE_URL=quay.io/kinvolk/kubelet
-          KUBELET_IMAGE_TAG=v1.19.4
+          KUBELET_IMAGE_TAG=v1.19.4-${os_arch}
```

Comment:
This is not needed. We publish arm64 compatible manfiests:
```console
$ docker manifest inspect quay.io/kinvolk/kubelet:v1.19.4
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1994,
         "digest": "sha256:91f6f7fca8e5c3bc466d855496563e13ad3bc115106227bd46145e3d4910110d",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1994,
         "digest": "sha256:3d74bf03a3d088ec04374148b087d895f309f3ae31b89e758239f54a2c157996",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
```

Reply:
I have no idea how this works :). I copied it from the packet module: https://github.com/kinvolk/lokomotive/blob/0b0c39c449f1b916182539f987de357e284b87a9/assets/terraform-modules/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl#L197

Reply:
Just remove the suffix and right image will be picked up automatically.